src/FundSourceApprover/FundSourceAllower/NativeFundSourceAllower.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.16;
+import "./FundSourceAllowerBase.sol";
+
+/// @notice Funds allower for a native currency.
+contract NativeFundSourceAllower is FundSourceAllowerBase {
```

Comment:
risk of fund loss if any token is sent here

Reply:
https://github.com/OffchainLabs/fund-distribution-contracts/pull/19#discussion_r1480067618